src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl registrationService;
+    private User registrateFirstUser;
+    private User registrateNotSameLoginUser;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        registrateFirstUser = new User();
+        registrateNotSameLoginUser = new User();
+        registrateFirstUser.setLogin("nafania");
+        registrateFirstUser.setPassword("oldDogBob");
+        registrateFirstUser.setAge(25);
+        registrateNotSameLoginUser.setLogin("nafaniaNotSame");
+        registrateNotSameLoginUser.setPassword("youngBob");
+        registrateNotSameLoginUser.setAge(20);

```

Comment:
u use registrateNotSameLoginUser only in one test method, so there is no need to do this logic before each